README.md

```diff
@@ -60,38 +60,36 @@ let segmenter = new Intl.Segmenter("fr", {granularity: "word"});
 let boundaries = segmenter.segment(input);
 let done = false;
 
-boundaries.index                // → 0
-boundaries.precedingSegmentType // → undefined
+boundaries.index                     // → 0
+boundaries.precedingSegmentType      // → undefined
 
-done = boundaries.following()   // → false
-boundaries.index                // → 6
-boundaries.precedingSegmentType // → "word" (describing "Allons")
+done = boundaries.following().atEnd  // → false
```

Comment:
It is still a bit weird this is called atEnd and not done. I need to understand this problem better.

Any other than that, the PR looks good.